server/config/config.js

```diff
@@ -0,0 +1,40 @@
+const path = require('path');
+const rootPath = path.normalize(__dirname + '/../..');
```

Comment:
What if the location of config.js file changes. We will then need to update the rootpath.

Is there anyway we can get the rootpath avoiding the hard-coding (/../..)